spec/support/examples/api.rb

```diff
@@ -21,3 +21,35 @@
     expect(response).to have_http_status(403)
   end
 end
+
+shared_examples 'unprocessable entity' do
+  it 'returns http unprocessable entity' do
+    # Some examples have a subject which needs to be called to make a request
+    subject if request.nil?
+
+    expect(response).to have_http_status(422)
+  end
+end
+
+shared_examples 'unauthorized for invalid token' do
```

Comment:
I lifted this up out of `filters_spec.rb` but also added a case for "without an Authorization header"